src/pages/resource/ReferencesPanel.tsx

```diff
@@ -0,0 +1,97 @@
+import { Field, Form, Formik } from 'formik';
+import { Select } from 'formik-material-ui';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { MenuItem } from '@material-ui/core';
+
+import Box from '../../components/Box';
+import TabPanel from '../../components/TabPanel/TabPanel';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import { ReferenceType, referenceTypeList } from '../../types/references.types';
+import { PublisherSearch } from './references/PublisherSearch';
+
+const StyledFieldWrapper = styled.div`
+  padding: 1rem;
+  flex: 1 0 40%;
```

Comment:
Apropos: Har du prøvd `styled(Field)\`padding: 1rem;\``? Prøvde å få det til å virke tidligere, men vet ikke hvorfor jeg ikke fikk det til... :(
ville unngått å bruke to tags for hver j* Field hvis det hadd virket.

Reply:
prøvde det nå, men det gikk ikke så bra nei. vi kan jo gjøre mer research på det etterhvert